src/common/entity/domain_icon.ts

```diff
@@ -51,7 +54,10 @@ export const domainIcon = (
       if (stateObj?.attributes.source_type === "router") {
         return compareState === "home" ? mdiLanConnect : mdiLanDisconnect;
       }
-      return mdiAccount;
+      if (stateObj?.attributes.source_type === "bluetooth") {
+        return compareState === "home" ? mdiBluetooth : mdiBluetoothOff;
```

Comment:
Don't like the `off`... Maybe
```suggestion
        return compareState === "home" ? mdiBluetoothConnect : mdiBluetooth;
```